src/components/pages/TeachersPage/index.tsx

```diff
@@ -156,6 +365,29 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
           {showDisplayedChat()}
         </Grid>
       </Grid>
+      <Grid container spacing={2} mt={2} pb={2}>
+        <Grid item xs={12}>
+          <Typography variant='h5' sx={{ color: 'white' }}>
+            Click to view full chat
+          </Typography>
+        </Grid>
+        {/* change to studentChats.map() to view actual chats  */}
+        {EXAMPLE_CHATS.map((chat, i) => {
+          // alter chat here to only have last 5 messages
```

Comment:
Love that the final version will show only the last several messages! Fantastic!